### ISSUE TITLE
Frame not filling side monitors

### ISSUE THREAD

**Issue by trapd00r** [jcs/sdorfehs#62]
I'm using the latest git version of sdorfehs, manjaro linux, xorg and a monitor
setup that looks like this:

```
[][  ][]
```

When I start sdorfehs, the active monitor is the primary monitor in the middle.
I open a terminal and it's all fine. 

Next, I switch to either of the side monitors, and open a terminal.
The terminal doesn't fill the entire screen because the frame doesn't fill the entire screen.
There's a gap between the frame and the upper edge of the screen, on both monitors.

If I execute `:only` in this frame, it's all good until next time I start xorg.

my xrandr looks like this:

```
HDMI-0 connected 1080x1920+3000+0 left (normal left inverted right x axis y axis) 510mm x 287mm
DP-0 connected 1080x1920+0+0 right (normal left inverted right x axis y axis) 510mm x 287mm
DP-2 connected primary 1920x1080+1080+0 (normal left inverted right x axis y axis) 598mm x 336mm
```

I also tried without a config, and the same thing happens.

Here's a screenshot to illustrate the problem (I ran ` xsetroot -solid '#ff0000'` first to make it easier to see the gap):

<img src="https://i.imgur.com/LGRdOU8.png" width="100" height="100">

**Comment by smemsh** [comment]
When you first start, after switching to the offending screen, but before using `:only`, what does `:sfdump` say?

**Comment by trapd00r** [comment]
Here goes:

`(frame :number 0 :x 0 :y 1142 :width 1080 :height 1858 :screenw 1080 :screenh 1920 :window 18874378 :last-access 59 :dedicated 0) 0,(frame :number 0 :x 1080 :y 1091 :width 1920 :height 1069 :screenw 1920 :screenh 1080 :window 0 :last-access 58 :dedicated 0) 1,(frame :number 0 :x 1080 :y 0 :width 1920 :height 1080 :screenw 1920 :screenh 1080 :window 0 :last-access 36 :dedicated 0) 2,(frame :number 0 :x 3000 :y 1142 :width 1080 :height 1858 :screenw 1080 :screenh 1920 :window 0 :last-access 0 :dedicated 0) 3`

**Comment by smemsh** [comment]
You describe three, but it shows four screens, do you know where the fourth one came from? (here edited for width and columns):
```
(fr 0 :x 0    :y 1142 :w 1080 :h 1858 :scrw 1080 :scrh 1920) 0,
(fr 0 :x 1080 :y 1091 :w 1920 :h 1069 :scrw 1920 :scrh 1080) 1,
(fr 0 :x 1080 :y 0    :w 1920 :h 1080 :scrw 1920 :scrh 1080) 2,
(fr 0 :x 3000 :y 1142 :w 1080 :h 1858 :scrw 1080 :scrh 1920) 3
```
I only use one screen myself, and I see only one screen in `sfdump`.  If you use three, I'd think there should be three shown in the output?

**Comment by trapd00r** [comment]
Yeah, sorry, I bought another monitor so I do have 4, but the issue still persists.
